src/main/presenter/devicePresenter/index.ts

```
@@ -276,6 +279,84 @@ export class DevicePresenter implements IDevicePresenter {
+
+        case 'all':
+          // 删除整个用户数据目录
+          removeDirectory(userDataPath)
+          break
+
+        default:
+          throw new Error(`Unknown reset type: ${resetType}`)
+      }
+
+      this.restartAppWithDelay()
+    } catch (error) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)